graph/resolver/user.resolvers.go

```diff
@@ -93,6 +94,63 @@ func (r *userResolver) Devices(ctx context.Context, obj *model.User) ([]*model.U
 	return resp, nil
 }
 
+func (r *userResolver) Discussions(ctx context.Context, obj *model.User) ([]*model.Discussion, error) {
+	authedUser := auth.GetAuthedUser(ctx)
```

Comment:
show discussions that user has access to (all public, flair matching, and user was accepted into)